public/documentation/samples/demoexpress.md

```diff
@@ -1,6 +1,6 @@
 # DemoExpress
 
-**DemoExpress** is a set of samples that show the Crosswalk advantage by having comprehensive API support: W3C standard APIs, embedded APIs, Cordova APIs and web runtime features. It includes popular web samples from Tizen and Android.
+**DemoExpress** is a set of samples that show the Crosswalk Project advantage by having comprehensive API support: W3C standard APIs, embedded APIs, Cordova APIs and web runtime features. It includes popular web samples from Android, Linux and etc.
 
```

Comment:
"Android, Linux and etc."  isn't quite right.  Do we have samples for Tizen or have those been dropped?  If not, we can just say "Android and Linux"    This is what I checked in.